### PR TITLE
Chore: Update travis.yml and Django settings to run tests with coverage.

### PR DESCRIPTION
**Pull request by Kyppy**
**Description**
This PR does the following:
- Installs 'coveralls', 'django_nose' and 'coverage' dependencies
- Update travis.yml to run manage.py tests and produce a 'coverage' report
- Updates Django 'settings.py' to include django_nose and configure coverage

### CODE REVIEW

**Review by ipaullly — DISMISSED** [review]
The changes seem up to standard

**Comment by LarryKarani** [comment]
Everything looks fine.

**Comment by LarryKarani** [comment]
Everything looks fine.

**Review by LarryKarani — DISMISSED** [review]
Looks fine

**Review by WNjihia — DISMISSED** [review]

**Review by MainaKamau92 — DISMISSED** [review]

**Review by Raywire — DISMISSED** [review]
Looks to be working alright.

File: .travis.yml [hunk 1]
```diff
@@ -10,11 +10,16 @@ addons:
...
   - pip install -r requirements.txt
 
 before_script: 
   - psql -c "CREATE DATABASE authors_haven;" -U postgres
 
+script:
+  - python manage.py test
+
+after_success:
+  - coveralls
```
Comment (kirega) [inline .travis.yml:25]: Remember the newline

**Review by ipaullly — APPROVED** [review]
everything looks to be in place

File: .travis.yml (hunk 1 above)
Comment (Kyppy) [reply .travis.yml:25]: Noted, thanks.

**Review by WNjihia — APPROVED** [review]

**Pull request merged**